docs/diagrams/FindCommandGetPredicateDiagram.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+!include style.puml
+
+participant ":FindCommandParser" as FindCommandParser LOGIC_COLOR
+participant ":XYZ Predicate" as Predicate LOGIC_COLOR
+
+mainframe sd get predicates
+    activate FindCommandParser
+    create Predicate
+    FindCommandParser -> Predicate
+    note right
+        Refers to any predicate with the corresponding prefix found in the ArgumentMultimap
+        For example, n/ in the command will give a NameContainsKeywordPredicate
+    end note
+    activate Predicate
+
+    Predicate --> FindCommandParser
+    deactivate Predicate
+
+    FindCommandParser -> FindCommandParser : add(XYZ Predicate)
```

Comment:
I think that maybe the FindCommandParser should be activated here again, since when calling a method from itself it should open another activation bar?